### ISSUE TITLE
Clarify what could cause session.keyStatuses to always be empty

### ISSUE THREAD

**Issue by XBDRX** [w3c/encrypted-media#468]
I'm implementing EME playback on a range of Smart TVs which support the spec "Encrypted Media Extensions W3C Working Draft 31 March 2015". The TVs have native DASH support so I am not using MSE.

I'm using commonly available test streams which I don't believe are using DASH features unsupported by the TVs:

**Working**
https://demo.unified-streaming.com/video/tears-of-steel/tears-of-steel-dash-playready.ism/.mpd

**Not Working**
https://media.axprod.net/TestVectors/v6.1-MultiDRM/Manifest_1080p.mpd

- Playready MediaKeySystemAccess is supported and media keys are created
- 'encrypted' events are received ok
- the license request appears to be successful
- the license response is sent to keySession.update 
- 'handlekeyStatusesChange' event is received but keyStatuses is *always* empty
- playback doesn't start, no errors are generated and no further video events are received

I've read through https://www.w3.org/TR/2015/WD-encrypted-media-20150331/ several times and it's not clear to me what can cause this to happen or what actions I should take.  Can someone please clarify?

**Comment by joeyparrish** [comment]
The spec you linked to is a 5-year-old working draft with a giant warning on it saying it's outdated:

![8MJoQUpRfynf4yN](https://user-images.githubusercontent.com/6843156/92528349-7ae33b00-f1dd-11ea-82de-adc5664751df.png)

But even in the current spec, I can think of no reason you should ever get empty key statuses.  My guess is that this would be a bug in the EME implementation of that smart TV, and I would recommend that you file a bug with the manufacturer.

Does this help?

**Comment by XBDRX** [comment]
Correct. That's the version of the EME spec implemented by LG WebOS 4.0 (2018) and WebOS 4.5 (2019) Smart TVs.

http://webostv.developer.lge.com/discover/specifications/supported-media-formats/

I will assume a bug in their implementation and discuss with them. I do notice that Shaka works on those devices so I will test those same streams using Shaka rather than my native player.

**Comment by Shakacro** [comment]
Hello. Sorry I don't have time to be here. Chains I carry long almost 15 years. I want to remove them because I don't know if everyone else has them too. Please help if you have time, before we donate them, today i dont know how is going, they are on Etherium, and there i ask for help and info. Btw I'm one off old crypto creator's. My biggest duty was Hacking for security. Before one month for end i make big hack for retiring this character. Got job in this company what i hacked few time's. And leaving everything that can help. If you have any questions ask me please. Love you all Shakacro.
